go/vt/vtgate/planbuilder/symtab.go

```diff
@@ -114,10 +114,10 @@ func (st *symtab) AddVindexTable(alias sqlparser.TableName, vindexTable *vindexe
 		c := &column{
 			origin: rb,
 			Vindex: cv.Vindex,
-			name:   cv.Column,
+			name:   cv.Columns[0],
```

Comment:
@sougou  - not sure what we should do here.

Reply:
This is the right thing to do for now. The symtab is used only for analyzing select statements, and `Columns[0]` is the right value to use here. This will get complicated when we start changing the `Map` functions.